media/css/firefox/sync.less

```diff
@@ -133,6 +137,18 @@
     }
 }
 
+.primary-buttons {
+    display: flex;
+    justify-content: space-around;
```

Comment:
Mixins aren't strictly needed here as this is shown to Firefox users only, but in #3765 I'm adding some just for general usefulness. Up to you if you would like use them here (happy to consolidate strategies, so to speak :smile:).